src/platform/unittest/DataPathTest.cpp

```diff
@@ -60,13 +60,15 @@ struct QuicAddr
     void Resolve(QUIC_ADDRESS_FAMILY af, const char* hostname) {
         CXPLAT_WORKER_POOL* WorkerPool = CxPlatWorkerPoolCreate(nullptr);
         CXPLAT_DATAPATH* Datapath = nullptr;
+        CXPLAT_DATAPATH_INIT_CONFIG InitConfig = {0};
```

Comment:
For the datapath tests to actually test DSCP, I think this needs to have the `DSCPRecvEnabled` field set to `TRUE`

Reply:
I can set that manually, but it looks like for this particular Resolve() test, it just initializes the datapath, runs CxPlatDatapathResolveAddress, then unintializes the datapath. I wasn't sure if it ran any of the DSCP stuff here.

Reply:
I just commented on the first occurrence in the file; looking more closely, it needs to be set on the second occurrence, since that's used for all the tests which actually check the DSCP value is sent/received.